systemtest/run-tests

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+#
+# run-tests - simple wrapper allowing shortcuts on invocation
+#
+
+# FIXME
+export SKOPEO_BINARY=${SKOPEO_BINARY:-/usr/bin/skopeo}
```

Comment:
(This bypasses the `SKOPEO_BINARY` logic in `helpers.bash`.)

Reply:
Removed - it was a holdover from earlier testing that I hadn't removed. Thanks for catching it.